services/madoc-ts/src/frontend/site/pages/view-canvas.tsx

```diff
@@ -223,6 +252,21 @@ export const ViewCanvas: React.FC<ViewCanvasProps> = ({
       });
   };
 
+  useEffect(() => {
+    if (manifestId) {
+      (async () => {
+        const manifest = await api.getManifestById(Number(manifestId));
+        if (manifest) setManifest(manifest);
+      })();
+    }
+    if (collectionId) {
+      (async () => {
+        const { collection } = await api.getCollectionById(Number(collectionId));
+        setCollectionName(collection?.label);
+      })();
+    }
+  }, [manifestId, collectionId]);
+
```

Comment:
Can you check the props in ViewCanvas - I believe the manifest and collection should be passed down.

Reply:
We have `  project,
  manifestUserTasks,
  canvasTask,
  userTasks,
  canUserSubmit,
  canvas,
  isLoadingTasks,` and the manifest id and collection id from the params

Reply:
Ah I don't think there added to the type
![Screenshot 2020-11-06 at 15 33 50](https://user-images.githubusercontent.com/8266711/98384289-799b8680-2045-11eb-83c9-d43145ce17ca.png)

Reply:
ahh, ok so I should be able to access it and update the type?